modules/Emails/include/ListView/ListViewDataEmailsSearchOnCrm.php

```diff
@@ -0,0 +1,316 @@
+                            $additionalDetailsFile = 'custom/modules/' . $this->lvde->seed->module_dir . '/metadata/additionalDetails.php';
+                        }
+                        require_once($additionalDetailsFile);
+                        $ar = $this->lvde->getAdditionalDetails($data[$dataIndex],
+                            (empty($this->lvde->additionalDetailsFunction) ? 'additionalDetails' : $this->lvde->additionalDetailsFunction) . $this->lvde->seed->object_name,
+                            $additionalDetailsEdit);
+                    }
+                    $pageData['additionalDetails'][$dataIndex] = $ar['string'];
+                    $pageData['additionalDetails']['fieldToAddTo'] = $ar['fieldToAddTo'];
+                }
+                next($rows);
+            }
+        }
+        $nextOffset = -1;
+        $prevOffset = -1;
+        $endOffset = -1;
+        if($count > $limit) {
+            $nextOffset = $offset + $limit;
+        }
+
+        if($offset > 0) {
+            $prevOffset = $offset - $limit;
+            if($prevOffset < 0)$prevOffset = 0;
+        }
+        $totalCount = $count + $offset;
+
+        if( $count >= $limit && $totalCounted){
+            $totalCount  = $this->lvde->getTotalCount($crmEmailsQuery);
+        }
+        SugarVCR::recordIDs($this->lvde->seed->module_dir, array_keys($idIndex), $offset, $totalCount);
+        $module_names = array(
+            'Prospects' => 'Targets'
+        );
+        $endOffset = (floor(($totalCount - 1) / $limit)) * $limit;
+        $pageData['ordering'] = $order;
+        $pageData['ordering']['sortOrder'] = $this->lvde->getReverseSortOrder($pageData['ordering']['sortOrder']);
+        //get url parameters as an array
+        $pageData['queries'] = $this->lvde->callGenerateQueries($pageData['ordering']['sortOrder'], $offset, $prevOffset, $nextOffset,  $endOffset, $totalCounted);
+        //join url parameters from array to a string
+        $pageData['urls'] = $this->lvde->callGenerateURLS($pageData['queries']);
+        $pageData['offsets'] = array( 'current'=>$offset, 'next'=>$nextOffset, 'prev'=>$prevOffset, 'end'=>$endOffset, 'total'=>$totalCount, 'totalCounted'=>$totalCounted);
+        $pageData['bean'] = array('objectName' => $seed->object_name, 'moduleDir' => $seed->module_dir, 'moduleName' => strtr($seed->module_dir, $module_names));
+        $pageData['stamp'] = $this->lvde->stamp;
+        $pageData['access'] = array('view' => $this->lvde->seed->ACLAccess('DetailView'), 'edit' => $this->lvde->seed->ACLAccess('EditView'));
+        $pageData['idIndex'] = $idIndex;
+        if(!$this->lvde->seed->ACLAccess('ListView')) {
+            $pageData['error'] = 'ACL restricted access';
+        }
+
+        $queryString = '';
+
+        if( isset($request["searchFormTab"]) && $request["searchFormTab"] == "advanced_search" ||
```

Comment:
Code inspection says that the conditions are confusing. You need to add ( condition ) instead.